test/unit/postprocessing/test_threshold_optimization.py

```diff
@@ -252,49 +266,49 @@ def test_threshold_optimization_equalized_odds(score_transform, y_transform,
     assert np.isclose(value_for_less_than_2_5,
-                      adjusted_predictor([sensitive_feature_names_ex1[0]], [2.499]))
-    assert base_value == adjusted_predictor([sensitive_feature_names_ex1[0]], [2.5])
-    assert base_value == adjusted_predictor([sensitive_feature_names_ex1[0]], [100])
+                      prob_pred([sensitive_feature_names_ex1[0]], [2.499]))
+    assert base_value == prob_pred([sensitive_feature_names_ex1[0]], [2.5])
+    assert base_value == prob_pred([sensitive_feature_names_ex1[0]], [100])
 
     # sensitive feature value B
     # p_ignore is the largest among the three classes indicating a large adjustment
     p_ignore = 0.1991991991991991
     base_value = prediction_constant * p_ignore
     value_for_less_than_0_5 = base_value + (1 - p_ignore) * 0.001
     assert np.isclose(value_for_less_than_0_5,
-                      adjusted_predictor([sensitive_feature_names_ex1[1]], [0]))
+                      prob_pred([sensitive_feature_names_ex1[1]], [0]))
     assert np.isclose(value_for_less_than_0_5,
-                      adjusted_predictor([sensitive_feature_names_ex1[1]], [0.5]))
+                      prob_pred([sensitive_feature_names_ex1[1]], [0.5]))
     assert base_value + 1 - \
-        p_ignore == adjusted_predictor([sensitive_feature_names_ex1[1]], [0.51])
+        p_ignore == prob_pred([sensitive_feature_names_ex1[1]], [0.51])
     assert base_value + 1 - \
-        p_ignore == adjusted_predictor([sensitive_feature_names_ex1[1]], [1])
+        p_ignore == prob_pred([sensitive_feature_names_ex1[1]], [1])
     assert base_value + 1 - \
-        p_ignore == adjusted_predictor([sensitive_feature_names_ex1[1]], [100])
+        p_ignore == prob_pred([sensitive_feature_names_ex1[1]], [100])
 
     # sensitive feature value C
     # p_ignore is 0 which means there's no adjustment
     p_ignore = 0
     base_value = prediction_constant * p_ignore
     value_between_0_5_and_1_5 = base_value + (1 - p_ignore) * 0.501
-    assert base_value == adjusted_predictor([sensitive_feature_names_ex1[2]], [0])
-    assert base_value == adjusted_predictor([sensitive_feature_names_ex1[2]], [0.5])
+    assert base_value == prob_pred([sensitive_feature_names_ex1[2]], [0])
+    assert base_value == prob_pred([sensitive_feature_names_ex1[2]], [0.5])
     assert np.isclose(value_between_0_5_and_1_5,
-                      adjusted_predictor([sensitive_feature_names_ex1[2]], [0.51]))
+                      prob_pred([sensitive_feature_names_ex1[2]], [0.51]))
     assert np.isclose(value_between_0_5_and_1_5,
-                      adjusted_predictor([sensitive_feature_names_ex1[2]], [1]))
+                      prob_pred([sensitive_feature_names_ex1[2]], [1]))
     assert np.isclose(value_between_0_5_and_1_5,
-                      adjusted_predictor([sensitive_feature_names_ex1[2]], [1.5]))
+                      prob_pred([sensitive_feature_names_ex1[2]], [1.5]))
```

Comment:
are all these changes related to plotting? Making each PR strictly limited to what it's supposed to do makes it easier to review and go back to see what's changed and why.

Reply:
Yes. If I don't make this adjustment then the tests fail. Root cause is that the threshold optimizer code used to be static functions as opposed to methods on the ThresholdOptimizer class. Now that we need to store some of the data during fit that needs to change which means that these tests needed a slight adjustment because the static functions are gone. It looks like a lot, but most of the changes here are added whitespace from indentation if you look at it side by side